### PR TITLE
[ME]: Fix misc issues with dashboard view

### PR DESCRIPTION
**Pull request by rcaplier**
### Description

This PR fix many issues:

- all collumns are now filled at loading
- the width of the list of record always takes the whole page
- several CSS fixes to match mockups (font size, weight, ...)
- draft badge fixed (number properly centered and invisble when no draft)
- pagination horizontaly centered

### Screenshots

image

### Quality Assurance Checklist

- [x] Commit history is devoid of any _merge commits_ and readable to facilitate reviews
- [x] If **new logic** ⚙️ is introduced: unit tests were added
- [x] If **new user stories** 🤏 are introduced: E2E tests were added
- [ ] If **new UI components** 🕹️ are introduced: corresponding stories in Storybook were created
- [ ] If **breaking changes** 🪚 are introduced: add the `breaking change` label
- [ ] If **bugs** 🐞 are fixed: add the `backport <release branch>` label
- [ ] The [documentation website](docs) 📚 has received the love it deserves

---

**This work is sponsored by **.

### CODE REVIEW

File: tailwind.base.css [hunk 1]
```diff
@@ -2,6 +2,10 @@
 @tailwind components;
 @tailwind utilities;
 
+:root {
+  --gn-ui-text-input-background-color: #f0f0f0;
+}
```
Comment (tkohr) [inline tailwind.base.css:7]: Where does this get applied? Shouldn't we rather move it to `.gn-ui-text-input` (if it's necessary)?

File: tailwind.base.css [hunk 2]
```diff
@@ -132,12 +136,12 @@
...
     --rounded: var(--gn-ui-badge-rounded, 0.25em);
-    --padding: var(--gn-ui-badge-padding, 0.375em 0.75em);
+    --padding: var(--gn-ui-badge-padding, 0.375em 0.875em);
     --text-color: var(--gn-ui-badge-text-color, var(--color-gray-50));
     --background-color: var(--gn-ui-badge-background-color, black);
     --opacity: var(--gn-ui-badge-opacity, 0.7);
-    @apply inline-block opacity-[--opacity] p-[--padding] rounded-[--rounded]
-      font-medium text-[length:0.875em] leading-none text-[color:--text-color] bg-[color:--background-color];
+    @apply opacity-[--opacity] p-[--padding] rounded-[--rounded]
+      font-medium text-[length:0.875em] text-[color:--text-color] text-xs bg-[color:--background-color] flex justify-center items-center content-center;
```
Comment (tkohr) [inline tailwind.base.css:144]: We should keep in mind that these changes affect all badges in gn-ui including the datahub.

File: tailwind.base.css (hunk 1 above)
Comment (rcaplier) [reply tailwind.base.css:7]: I forgot to remove it.

File: tailwind.base.css (hunk 2 above)
Comment (rcaplier) [reply tailwind.base.css:144]: Yes, I checked the datahub and badges seems ok.

**Comment by jahow** [comment]
@tkohr did some more tests and found these suggestions:
* could we have at least an empty bar on top of the my-drafts screen? otherwise the layout jumps when navigating to this page
* could we have the notifications container as part of the Dashboard page component and not of individual pages?
* also the scroll issue on the dashboard page is still there :)

**Comment by tkohr** [comment]
Thanks for the review @jahow! I've addressed the points above, hoping it looks good to merge now :-)

**Comment by tkohr** [comment]
> @tkohr wait, before merging could you please change back the text size of the gn-ui-badge to what it was before? and then use a smaller text size in the editor when needed? otherwise the keywords look too small in the datahub now

Done. Note that I had to make the `font-size` configurable as well. Couldn't get it done with `[style.font-size]="'12px'"`. I also noticed that the format tag (that does not use the badge) did not use the right font size and adapted this along the way, oriented on other mockup badges in the table as I did not see any for formats.

**Review by jahow — APPROVED** [review]
Great work, thanks to both of you @tkohr and @rcaplier! I'm approving, feel free to merge when you think it's ready

**Pull request merged**